base.scss

```diff
@@ -143,7 +143,7 @@
 @import "./src/stories/Blocks/advanced-search/advanced-search";
 @import "./src/stories/Blocks/article/article";
 @import "./src/stories/Blocks/event-page/event-page";
-@import "./src/stories/Blocks/event-list-page/event-list-page";
+@import "./src/stories/Blocks/list-page/list-page";
```

Comment:
Please reconsider the name here.

I understand the intention but `list-page` is *very* generic. It would be nice if it was used across all lists like the search result and perhaps also other places, but that is not the case at the moment anyway.

How about something like `content-list-page`? `editorial-content-list-page` is quite lengthy...

Reply:
I'll just go with `content-list-page`